File: Mandatory/023 - Trees and Graphs - How Tall is the Tree/BinaryTreeHeight.js
function binaryTreeHeight(arrayTree) {
  let level = 1;
  while (arrayTree.length > (Math.pow(2, level)) - 1) {
    level += 1
  }

  return level;
}

console.log(binaryTreeHeight([2, 7, 5, 2, 6, 0, 9]));
// => 3

File: Mandatory/023 - Trees and Graphs - How Tall is the Tree/BinaryTreeHeight.py
def binary_tree_height(array_tree):    
  level = 1
  while (len(array_tree) > (2 ** level) - 1):
    level += 1
    
  return level

print(binary_tree_height([2, 7, 5, 2, 6, 0, 9]))
# => 3